presto-spi/src/main/java/com/facebook/presto/spi/HostAddress.java

```diff
@@ -73,7 +77,8 @@
      */
     private final int port;
 
-    private HostAddress(String host, int port)
+    @ThriftConstructor
+    public HostAddress(String host, int port)
```

Comment:
Had to make this a public constructor since drift did not have support for factory methods like Jackson (We can add in something if there are more use cases that require this). At present drift only supports constructors and Builder method . Let me know if there is a better way.

Reply:
I think it's fine.